pkr/kard.py

```diff
@@ -197,29 +197,30 @@ def set_meta(kard, extra):
         merge(kard.driver.get_meta(extra, kard), kard.meta)
 
         # Prevent duplication of features in case of update
-        kard.meta['features'] = list(set(kard.meta['features']))
+        kard.meta['features'] = list(kard.meta['features'])
```

Comment:
in that case, you can remove the whole line...

Reply:
Done !